sdk/go/common/workspace/plugins_test.go

```diff
@@ -235,6 +235,49 @@ func TestPluginSelection_EmptyVersionWithAlternatives(t *testing.T) {
 	assert.Equal(t, "0.2.0", result.Version.String())
 }
 
+func TestPluginDownloadUrl(t *testing.T) {
```

Comment:
I'd love to see this as a table driven test

Reply:
I will be looping back to this code once we make it the default so I will fix these up then just incase it needs to change